relentless/simulate/simulate.py

```diff
@@ -736,11 +754,103 @@ class InitializeRandomly(GenericOperation):
     masses : dict
         Masses of each particle type. Defaults to None, which means particles
         have unit mass.
+    diameters : dict
+        Diameter of each particle type. Defaults to None, which means particles
+        are randomly inserted without checking their sizes.
 
     """
-    def __init__(self, seed, N, V, T=None, masses=None):
+    def __init__(self, seed, N, V, T=None, masses=None, diameters=None):
         super().__init__(seed, N, V, T, masses)
```

Comment:
super().__init__(seed, N, V, T, masses, diameters)

Reply:
Oh whoops! This is a big mistake, thanks for spotting it! I think the other code path you marked was probably never taken because it was defaulting the diameters to `None`. Otherwise, it should have been an error to miss the diameter argument. Let me take a look at the unit tests to see if we can add something to make sure this triggers.